README.md

```diff
@@ -52,8 +52,8 @@ git clone https://github.com/ethereum/remix-project.git
 cd remix-project
 npm install
 npm run build:libs // Build remix libs
-nx build
-nx serve
+npm run build
+npm run serve
```

Comment:
`npm run build` builds the whole project and limits the available command/flags that a user can leverage when using `nx`.
Example: `nx build remix-simulator` or `nx build --skip-nx-cache`

`npm run build` will only be useful if we want to build the whole project. I think we should still keep the old changes or provide a better description to distinguish the usecase of having `nx` installed global compared to using the local `nx` within the `node_modules`.